recipes/mosdef_cassandra/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "mosdef_cassandra" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88e87ee6cbc266e43d169e433e010789b8ccdc1552f046afd94db8f9c75ed771" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MaginnGroup/mosdef_cassandra/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<35]
```

You might be overriding the first `skip` when you define the second `skip`. Try:
```suggestion
  skip: True  # [win or py<35]
```